.github/workflows/crowdin.yml

```
@@ -0,0 +1,35 @@
+name: Crowdin Action
+
+on:
+  push:
+    branches: [ master ] 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)